src/main/java/wooteco/subway/auth/infrastructure/AuthorizationExtractor.java

```diff
@@ -5,8 +5,8 @@
 
 public class AuthorizationExtractor {
     public static final String AUTHORIZATION = "Authorization";
-    public static String BEARER_TYPE = "Bearer";
     public static final String ACCESS_TOKEN_TYPE = AuthorizationExtractor.class.getSimpleName() + ".ACCESS_TOKEN_TYPE";
+    public static String BEARER_TYPE = "Bearer";
```

Comment:
```suggestion
    public static final String BEARER_TYPE = "Bearer";
```

제공된 코드이긴 하지만 이런 부분도 체크해 주세요~

Reply:
넵 반영했습니다~